src/traits.rs

```diff
@@ -1498,6 +1503,11 @@ pub trait ComponentExt: Debug {
     ///     println!("{}°C", component.max());
     /// }
     /// ```
+    ///
+    /// ## Linux hwmon
+    ///
+    /// Read current Maximum value encountered may be computed by sysinfo
+    /// or from kernel in case of an errors, returns `f32::NAN`.
```

Comment:
```suggestion
    /// ## Linux
    ///
    /// Returns `f32::NAN` if an error occurred.
```

Reply:
As a lib-user I would prefers knowing if it's computed or read instead of doing `strace` but okay.

Reply:
Tried to found a common ground on about to explain the behaviour without being too verbose.